doc/whats_new.rst

```diff
@@ -19,7 +19,7 @@ Current
 Changelog
 ~~~~~~~~~
 
-- Nothing yet
+- Add support for metadata in :class:`mne.Epochs` by `Chris Holdgraf`_, `Jona Sassenhagen`_, and `Eric Larson`_
```

Comment:
we should add @agramfort in there as well...he and I mapped out the original prototypes for this at SciPy 2017!